test/onlynerByTest.js

```diff
@@ -0,0 +1,16 @@
+describe('Open Page', function() {
+        it('Open browser page in url and input login & password', async function () {
+        await browser.url('https://www.onliner.by/');
+        await browser.pause(5000)
+        const enterButton = $('//div[text()=\'Вход\']');
+        await browser.pause(3000);
+        enterButton.click();
+        await browser.pause(3000);
+        let inputMail = $('//input[@placeholder = \'Ник или e-mail\']');
+        inputMail.addValue('test@gmail.com');
+        await browser.pause(3000);
+        let inputPassword = $('//input[@placeholder = \'Пароль\']');
+        inputPassword.addValue('12314351325')
```

Comment:
Create object like 

`const user= { username: 'test!gmail.com', password: '123455' }
`

And use instead of hardcoded strings

Reply:
Also all strings should be as constants